src/components/Expander/Expander.jsx

```diff
@@ -138,6 +171,7 @@ const Expander = createClass({
 							<span key={this._labelKey}>{labelChildProp.children}</span>
 						: null}
 					</ReactCSSTransitionGroup>
+					{withEmphasis ? (<ChevronButton isExpanded={isExpanded} />) : null}
```

Comment:
if the only difference between normal and emphasis is the order of the elements, it can be done in the css with `flex-direction: reverse`